src/axom/quest/detail/MarchingCubesPartParallel.hpp

```diff
@@ -867,6 +880,61 @@ class MarchingCubesImpl : public MarchingCubesSingleDomain::ImplBase
 
   double m_contourVal = 0.0;
 };
+
+static std::unique_ptr<axom::quest::MarchingCubesSingleDomain::ImplBase>
+newMarchingCubesPartParallel(MarchingCubesRuntimePolicy runtimePolicy, int dim)
+{
+  using ImplBase = axom::quest::MarchingCubesSingleDomain::ImplBase;
+  using RuntimePolicy = axom::quest::MarchingCubesRuntimePolicy;
+
+  SLIC_ASSERT(dim >= 2 && dim <= 3);
+  std::unique_ptr<ImplBase> impl;
+  if(runtimePolicy == RuntimePolicy::seq)
+  {
+    impl = dim == 2
+      ? std::unique_ptr<ImplBase>(
+          new MarchingCubesPartParallel<2, axom::SEQ_EXEC, axom::SEQ_EXEC>)
+      : std::unique_ptr<ImplBase>(
+          new MarchingCubesPartParallel<3, axom::SEQ_EXEC, axom::SEQ_EXEC>);
+  }
+  #ifdef _AXOM_MARCHINGCUBES_USE_OPENMP
```

Comment:
Should this be converted to the `AXOM_RUNTIME_POLICY_USE_{OPENMP,CUDA,HIP}` defines that you recently introduced?

See:
* https://github.com/LLNL/axom/blob/631e5ef14952ab2a982498c0d31f972724155c23/src/axom/core/execution/runtime_policy.hpp#L44-L55
* https://github.com/LLNL/axom/pull/1225

Reply:
Yes.  Thanks.